css/main.css

```diff
@@ -1,3 +1,276 @@
+  width: 100%;
+  color: var(--red-brown);
+}
+
+.login,
+nav {
+  display: none;
+}
+
+.search {
+  border: 1px var(--red-brown);
+  background-color: var(--white);
+  justify-content: space-around;
+  width: 20px;
+}
+
+.search-input {
+  grid-column: 1 / 3;
+  grid-row: 1 / 2;
+}
+
+.shopping-cart {
+  grid-column: 3 / -1;
+  grid-row: 1 / 2;
+  width: 50px;
+}
+.shopping {
+  width: 100%;
+}
+
+.hamburger {
+  grid-column: 3 / -1;
+  grid-row: 2;
+  width: 50px;
+  margin-top: 10px;
+}
+
+.header-text {
+  grid-column: 1 / 3;
+  grid-row: 3 / -2;
+  margin: auto;
+  padding: 10px;
+  color: var(--red-brown);
+}
+
+.main {
+  width: 100%;
+  background-size: cover;
+  text-align: center;
+  position: relative;
+}
+
+.welcome img {
+  position: relative;
+  margin-top: 6rem;
+}
+
+.esbt {
+  position: absolute;
+  left: 35px;
+  top: 22rem;
+  color: var(--white);
+  font-size: 1.7rem;
+  border-bottom: 4px solid;
+}
+
+.first-icon {
+  margin: auto 10px;
+  width: 100%;
+}
+
+.year {
+  position: absolute;
+  color: var(--white);
+  left: 347px;
+  top: 22rem;
+  font-size: 1.7rem;
+  border-bottom: 4px solid;
+}
+.prickles {
+  color: var(--green-black);
+  left: 35px;
+  font-size: 3rem;
+  text-align: center;
+  padding-bottom: 1.5px;
+}
+
+.indoor {
+  position: absolute;
+  color: var(--white);
+  left: 150px;
+  top: 450px;
+  text-align: center;
+  font-size: 1rem;
+}
+
+.shop {
+  background-color: var(--red-brown);
+  color: var(--white);
+  position: absolute;
+  border: none;
+  align-items: center;
+  top: 535px;
+  left: 200px;
+}
+
+.cacti {
+  color: var(--red-brown);
+  border: none;
+  align-items: center;
+  font-size: 1.7rem;
+}
+.plants {
+  color: var(--red-brown);
+  border: none;
+  align-items: center;
+  font-size: 1.7rem;
+}
+.succulent {
+  color: var(--red-brown);
+  border: none;
+  align-items: center;
+  font-size: 1.7rem;
+}
+
+.paragraph {
+  color: var(--red-brown);
+  margin-bottom: 40px;
+}
+.sale {
+  background-color: var(--green-black);
+  color: var(--white);
+  font-size: 3.5rem;
+  text-align: center;
+  height: 25rem;
+}
+.sale-btn {
+  font-size: 25px;
+  width: 12rem;
+}
+
+.form-container {
+  background-color: var(--white);
+  margin: 7%;
+  flex-direction: column;
+  font-style: normal;
+  border: none;
+}
+.orders {
+  color: var(--green-black);
+  font-size: 1.7rem;
+  text-align: center;
+  margin: 18px;
+  padding: 45px;
+  border-bottom: 4 px solid;
+}
+.name,
+.lastname,
+.email,
+.phone {
+  display: block;
+  margin-bottom: 40px;
+  width: 500 px;
+  height: 40 px;
+}
+
+button {
+  background-color: var(--red-brown);
+  color: var(--white);
+  border: none;
+}
+.form {
+  color: var(--red-brown);
+  font-size: 1rem;
+}
+
+.footer {
+  background-color: var(--green-black);
+  color: var(--white);
+}
+
+.info-footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.info {
+  margin-top: 5px;
+  padding: 5px;
+}
```

Comment:
This is good! You are re-using the `info` class for each block, and setting flex rules on the container (`info-footer`) to center the children - good work 👌